install/kubernetes/helm/istio/charts/mixer/templates/deployment.yaml

```diff
@@ -32,6 +32,12 @@
         volumeMounts:
         - name: uds-socket
           mountPath: /sock
+        livenessProbe:
+          httpGet:
+            path: /metrics
```

Comment:
how about using `/version` instead? the handler always prints the version string, 0 computational cost...